packages/backend/src/controllers/projectController.ts

```diff
@@ -798,6 +799,32 @@ export class ProjectController extends BaseController {
 
     @Middlewares([allowApiKeyAuthentication, isAuthenticated])
     @SuccessResponse('200', 'Success')
+    @Put('{projectUuid}/tags/yaml')
+    @OperationId('replaceYamlTags') // TODO: should this be renamed to upsert?
```

Comment:
I think replace makes sense `replace` to be in the api docs, upsert is something more about the technical implementation.